lib/actions/deploy.coffee

```diff
@@ -31,11 +31,21 @@ parseInput = Promise.method (params, options) ->
 
 	return [appName, options.build, source, image]
 
+# Builds and returns a Docker-like progress bar like this:
+# [==================================>               ] 64%
+renderProgress = (percentage, stepCount = 50) ->
+	_ = require('lodash')
+	percentage = Math.max(0, Math.min(percentage, 100))
+	barCount = stepCount * percentage // 100
+	spaceCount = stepCount - barCount
+	bar = "[#{_.repeat('=', barCount)}>#{_.repeat(' ', spaceCount)}]"
+	return "#{bar} #{percentage.toFixed(1)}%"
```

Comment:
This is now our third separate progress bar implementation. We've got one in the [docker-progress](https://github.com/resin-io/docker-progress/blob/master/index.coffee) that's almost but not _exactly_ the same as this, one in [resin-cli-visuals](https://github.com/resin-io-modules/resin-cli-visuals/blob/master/lib/widgets/progress.coffee), and this one.

It'd be nice to be consistent, and use one progress bar everywhere. Can we use the resin-cli-visuals bar we already use in the CLI [for image downloads](https://github.com/resin-io/resin-cli/blob/dede4bb3293a867f2dfdeabe0c9492ebff1acf7f/lib/actions/os.coffee#L111) for this too?

In a perfect world, I'd also quite like to change the UI to use that too, and have a standard resin style for our CLI interfaces (since Etcher also uses this one), but one step at a time :-).